core/serialisation/data_serialiser.py

```diff
@@ -88,6 +112,13 @@ def serialise_download_data(
     Each extended query and its corresponding schema should be added to the table_queries object in order to be
     serialised. Each additional query uses the base_query parameter as a starting point.
 
+    Fields serialised from a JSONB use custom serialisation classes in which dump_default=None.
```

Comment:
could you move this bit to the module docstring, and make sure the "guidance" in there is up to date?